vespa/datasets/utils.py

```
@@ -1,4 +1,4 @@
-from torch import Tensor, empty, float32, int64, tensor
+from torch import Tensor, empty, float32, int64, tensor, zeros
```

Aqui tu tas usando Tensor e tensor.
Cuidado que os dois tem um resultado parecido, mas algumas vezes, da muito errado (silent error)

---

O Tensor só é utilizado para declarar o tipo da imagem quando a função é chamada. Todas as tranformações são feitas com tensor.